examples/sparse_regression.py

```diff
@@ -43,7 +44,7 @@ def dot(X, Z):
 
 
 # The kernel that corresponds to our quadratic regressor.
-def kernel(X, Z, eta1, eta2, c, jitter=1.0e-6):
+def kernel(X, Z, eta1, eta2, c, jitter=1.0e-4):
```

Comment:
need more jitter because the prior now has more mass at sigma=0

Reply:
Did you face Cholesky issues or `nan` during sampling? I think we addressed it by setting `energy=inf` but if it happens with this example, I need to take a look again.

Reply:
no sampling was fine; it's more that some downstream predictive stuff was less stable